statesync/mocks/state_provider.go

```diff
@@ -6,6 +6,7 @@ import (
 	context "context"
 
 	mock "github.com/stretchr/testify/mock"
+
```

Comment:
This format change thing is a bit annoying, I got it as well when running `make mockery` then `make format` - ideally these files shouldn't be changing after they generate. I'm guessing we need to fix a specific version of `goimports` or smth, will look at upstream